src/Ocelot.Provider.Kubernetes/Ocelot.Provider.Kubernetes.csproj

```diff
@@ -29,8 +29,8 @@
     <Compile Remove="KubeApiClientFactory.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="KubeClient" Version="2.4.10" />
-    <PackageReference Include="KubeClient.Extensions.DependencyInjection" Version="2.4.10" />
+    <PackageReference Include="KubeClient" Version="2.5.8" />
+    <PackageReference Include="KubeClient.Extensions.DependencyInjection" Version="2.5.8" />
```

Comment:
### TODO
The package has been updated to version [2.5.10](https://www.nuget.org/packages/KubeClient/#versions-body-tab), released on August 11, 2024. It is necessary to update to this version prior to the next release.

Reply:
The packages was updated in release [23.3.4](https://github.com/ThreeMammals/Ocelot/releases/tag/23.3.4) on Oct 3, 2024